.github/workflows/gradle-build-pr.yml

```diff
@@ -21,6 +21,9 @@ on:
       - 'datarepo-clienttests/**'
       - '.github/workflows/gradle-build-pr.yml'
       - '.swagger-codegen-ignore'
+  workflow_dispatch: {}
+  schedule:
+    - cron: '0 4 * * *' # run at 4 AM UTC, 12PM EST.
```

Comment:
Is there anyway to test that this will work as a nightly run against develop? I feel pretty confident it will work, but I don't know how to test without merging it. @smark88 ?

Reply:
Thoughts about doing this every `n` hours as per the original ticket instead?

Reply:
I think it's probably fine to start with this and we can observe.  One question: do we need to put a timeout?  Just to make sure we don't get colliding runs?  That could end up eating all of our integration instances

Reply:
I also wonder if at some point we should rename this action since it's not really a PR check anymore...

Reply:
Run frequency: Good question @fboulnois. I thinking starting once a night is also good. 

Timeouts:  We have a timeout on some of the individual steps that are at risk of running long.  I think we saw things running for extremely long yesterday b/c it was getting stuck on steps that should never take more than a couple of minute (like checking out code). 

Name: Renamed.

Reply:
lol...most efficient comment ever